plugins/GSdx/Window/GSLinuxDialog.cpp

```diff
@@ -600,6 +602,7 @@ void populate_osd_table(GtkWidget* osd_table)
 	InsertWidgetInTable(osd_table , opacity_label      , opacity_slide);
 	InsertWidgetInTable(osd_table , log_timeout_label  , log_timeout_text);
 	InsertWidgetInTable(osd_table , max_messages_label , max_messages_spin);
+	InsertWidgetInTable(osd_table , monitor_pos_label  , monitor_pos_combo);
```

Comment:
Forgot to move them on gtk.

Reply:
Just got occupied, changes are pushed now.